### DISCUSSION TITLE
Does deno have a functional module like node's http.agent?

### DISCUSSION THREAD

**Discussion by leekeal** [denoland/deno#11178] [Q&A]
Does deno have a functional module like nodejs http.agent? We want to develop a Reverse Proxy，
Hope to achieve keepAlive and other features。

**Comment by lucacasonato** [comment]
I am not sure if this is exactly what you are looking for, but you can create custom HTTP clients using the unstable [`Deno.createHttpClient` API](https://doc.deno.land/builtin/unstable#Deno.createHttpClient). Keep alive is automatic for both the global client using `fetch`, and custom clients.
